### PR TITLE
Update release notes for v1.4.0

### PR DESCRIPTION
**Pull request by mheon**
For real this time.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,7 +2,36 @@
...
+### Misc
+- The `podman commit` command is now usable with the Podman remote client
+- The `--signature-policy` flag (used with several image-related commands) has been deprecated
+- The `podman unshare` command now defines two environment variables in the spawned shell: `CONTAINERS_RUNROOT` and `CONTAINERS_GRAPHROOT`, pointing to temporary and permanent storage for rootless containers
+- Updated vendored containers/storage and containers/image libraries with numerous bugfixes
+- Updated vendored Buildah to v1.8.3
+- Podman now requires Conmon v0.2.0
+- The `podman cp` command is now aliased as `podman container cp`
+- Rootless Podman will now default `init_path` using root Podman's configuration files (`/etc/containers/libpod.conf` and `/usr/share/containers/libpod.conf`) if not overridden in the rootless configuration
+>>>>>>> Update release notes for v1.4.0
```
Comment (haircommander) [inline RELEASE_NOTES.md]: rebase headaches?

Comment (mheon) [reply RELEASE_NOTES.md]: Fixed

**Comment by mheon** [comment]
Alright. Looks solid now. Hitting the button.

**Comment by haircommander** [comment]
/lgtm
go ahead and hit that naughty red button

**Pull request merged**